### PR TITLE
[FAB-124] Participant support

### PR DESCRIPTION
**Pull request by daystram**
- [x] Help and done buttons
- [x] Help and done notifications
- [x] Hub participant toolbar
- [x] Tag input 
- [x] Layout and color tweaks

### CODE REVIEW

File: fableous-fe/src/containers/HubCanvasPage.tsx [hunk 1]
```diff
@@ -123,15 +147,34 @@ export default function HubCanvasPage() {
...
-                msg.data as WSControlMessageData;
+              const {
+                classroomToken: classroomTokenFromWs,
+                help,
+                done,
+              } = msg.data as WSControlMessageData;
               if (classroomTokenFromWs) {
                 setClassroomToken(classroomTokenFromWs);
               }
+              console.log(msg);
```
Comment (nicolauscg) [inline fableous-fe/src/containers/HubCanvasPage.tsx:162]: remove console.log if unintentional

Comment (daystram) [reply fableous-fe/src/containers/HubCanvasPage.tsx]: done

**Review by nicolauscg — COMMENTED** [review]
all good for changes outside the recent commits

File: fableous-fe/src/components/canvas/Canvas.tsx [hunk 2]
```diff
@@ -934,7 +934,7 @@ const Canvas = forwardRef<ImperativeCanvasRef, CanvasProps>(
           style={{
             position: "absolute",
             width: offsetWidth,
-            borderRadius: "30px",
+            borderRadius: "24px",
```
Comment (nicolauscg) [inline fableous-fe/src/components/canvas/Canvas.tsx:937]: if intentional also change in CursorScreen, and div with classname `bg-white` in hub and controller page

File: fableous-fe/src/containers/HubCanvasPage.tsx [hunk 3]
```diff
@@ -608,28 +694,70 @@ export default function HubCanvasPage() {
...
+                    <Chip label={tag} color="secondary" />
+                  ))}
+                </div>,
               ]}
             />
           </Grid>
         </Grid>
-        <Grid container className="flex-1 my-4">
-          <Grid item xs={12}>
+        <Grid container spacing={2} className="flex-1 my-4">
```
Comment (nicolauscg) [inline fableous-fe/src/containers/HubCanvasPage.tsx:712]: it shows a scrollbar on my localhost, apply styling similar to hideScrollbar in CanvasToolbar

also do we want to limit the width as well? CanvasToolbar currently has max width 80px

image

**Review by nicolauscg — COMMENTED** [review]
all good aside minor comments

File: fableous-fe/src/containers/HubCanvasPage.tsx (hunk 3 above)
Comment (daystram) [reply fableous-fe/src/containers/HubCanvasPage.tsx:712]: maxWidth for layer toolbar was initially set to 100. will revert

File: fableous-fe/src/components/canvas/Canvas.tsx (hunk 2 above)
Comment (daystram) [reply fableous-fe/src/components/canvas/Canvas.tsx:937]: got it

**Review by nicolauscg — APPROVED** [review]

**Pull request merged**